### PR TITLE
Build with CircleCI

### PR DESCRIPTION
**Pull request by pastak**
- build package in circle.ci
- push repository on packagecloud.io

It make installing easy.

You can install this commands 

    $ curl -s https://packagecloud.io/install/repositories/gyazo/gyazo-for-linux/script.deb.sh | sudo bash
    $ sudo apt-get install gyazo

Merge into `release` branch will trigger build packages and release.

### CODE REVIEW

**Comment by pastak** [comment]
@rakusai @drobune @gyazo/owners Please review it :smile:

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ http://gyazo.com/
 
 Install Ruby and ImageMagick before installing Gyazo.
 
-    $ sudo apt-get install ruby imagemagick
-    $ sudo dpkg -i gyazo_XXX_all.deb
+    $ curl -s https://packagecloud.io/install/repositories/gyazo/gyazo-for-linux/script.deb.sh | sudo bash
```
Comment (drobune) [inline README.md]: i want to add `$ sudo apt-get update` before install

Comment (pastak) [inline README.md]: :+1:

**Comment by pastak** [comment]
I write manually supported OS and Versions because `.deb` requires os and version in packegecloud

**Comment by drobune** [comment]
[Gyazo](https://gyazo.com/0bf79999bfaf95368f59cfc5311e7902)

**Comment by pastak** [comment]
:beers:

**Pull request merged**